recipe/meta.yaml

```diff
@@ -12,89 +12,160 @@ source:
+        - anytree >=2.8.0
+        - casadi >=3.6.7
+        - posthog >=3.6.5
+        - numpy >=1.23.5,<2.0.0
+        - pandas >=1.5.0
+        - pooch >=1.8.1
+        - scipy >=1.11.4
+        - sympy >=1.12
+        - typing-extensions >=4.10.0
+        - xarray >=2022.6.0
+        - pyyaml
+        - platformdirs
+
+    test:
+      imports:
+        - pybamm
+        - pybamm.discretisations
+        - pybamm.experiment
+        - pybamm.experiment.step
+        - pybamm.expression_tree
+        - pybamm.expression_tree.operations
+        - pybamm.expression_tree.printing
+        - pybamm.geometry
+        - pybamm.meshes
+        - pybamm.models
+        - pybamm.models.full_battery_models
+        - pybamm.models.full_battery_models.equivalent_circuit
+        - pybamm.models.full_battery_models.lead_acid
+        - pybamm.models.full_battery_models.lithium_ion
+        # - pybamm.models.full_battery_models.lithium_metal
+        - pybamm.models.submodels
+        - pybamm.models.submodels.active_material
+        - pybamm.models.submodels.convection
+        - pybamm.models.submodels.convection.through_cell
+        - pybamm.models.submodels.convection.transverse
+        - pybamm.models.submodels.current_collector
+        - pybamm.models.submodels.electrode
+        - pybamm.models.submodels.electrode.ohm
+        - pybamm.models.submodels.electrolyte_conductivity
+        - pybamm.models.submodels.electrolyte_conductivity.surface_potential_form
+        - pybamm.models.submodels.electrolyte_diffusion
+        - pybamm.models.submodels.equivalent_circuit_elements
+        - pybamm.models.submodels.external_circuit
+        - pybamm.models.submodels.interface
+        - pybamm.models.submodels.interface.interface_utilisation
+        - pybamm.models.submodels.interface.kinetics
+        - pybamm.models.submodels.interface.kinetics.inverse_kinetics
+        - pybamm.models.submodels.interface.lithium_plating
+        - pybamm.models.submodels.interface.open_circuit_potential
+        - pybamm.models.submodels.interface.sei
+        - pybamm.models.submodels.oxygen_diffusion
+        - pybamm.models.submodels.particle
+        - pybamm.models.submodels.particle_mechanics
+        - pybamm.models.submodels.porosity
+        - pybamm.models.submodels.thermal
+        - pybamm.models.submodels.thermal.pouch_cell
+        - pybamm.models.submodels.transport_efficiency
+        - pybamm.parameters
+        - pybamm.plotting
+        - pybamm.solvers
+        - pybamm.solvers.c_solvers
+        - pybamm.spatial_methods
+      requires:
+        - pip
+        - python {{ python_min }}
+      commands:
+        - pip check
+
+  - name: pybamm
+    build:
+      noarch: python
+
+    requirements:
+      host:
+        - pip
+        - python {{ python_min }}
+        - setuptools >=64
+      run:
+        - {{ pin_subpackage('pybamm-base', exact=True) }}
+        # not available on conda-forge
+        # - bpx
```

Comment:
Didn't know this – we should try to package it. @prady0t, would you like to help out? We can get a few more maintainers from BPX itself added later on (it is the case for many feedstocks that maintainers don't fully overlap with the upstream project)

The docs for adding a new package are here: https://conda-forge.org/docs/maintainer/adding_pkgs/

Reply:
Already on the way - https://github.com/conda-forge/staged-recipes/pull/28431 😉